### PR TITLE
Add inline branch counts for evennia/contrib/dice.py, close #5

### LINKED ISSUE #5: Branch coverage method func@158-251@./evennia/contrib/dice.py
Implement branch coverage by manual instrumentation of the source code for ten functions with high cyclomatic complexity.
Again, for small groups, the number goes down proportionally to eight functions for groups with four members, etc.

Use a separate development branch on git for this, as this code is not permanent.

The simplest method for this is as follows:

Identify all branches in the given functions; assign a unique number (ID) to each one.
Before the program starts (before the first instruction in "main" or as the first step in the unit test harness), create data structures that hold coverage information about specific branches.
Before the first statement of each branch outcome (including to-be-added "else" clauses if none exist), add a line that sets a flag if the branch is reached.
At the end of the program (as the last instruction in "main" or at the end of all unit tests), write all information about the taken branches taken to a file or console.

### PR DESCRIPTION
**Pull request by LukasSzerszen**

### CODE REVIEW

**Comment by Plongy** [comment]
Instead of doing the PR towards master, do it towards my branch

File: evennia/contrib/dice.py [hunk 1]
```diff
@@ -156,9 +157,11 @@ class CmdDice(default_cmds.MuxCommand):
...
 
     def func(self):
+        branchdata = branchlogger.branchdata["evennia/contrib/dice.py"]["CmdDice/func"]
         """Mostly parsing for calling the dice roller function"""
 
         if not self.args:
+            branchdata[0].append(0)
             self.caller.msg("Usage: @dice <nr>d<sides> [modifier] [conditional]")
             return
         argstring = "".join(str(arg) for arg in self.args)
```
Comment (Plongy) [inline evennia/contrib/dice.py]: You are missing the `else`-branch here

**Comment by LukasSzerszen** [comment]
Ok, Shall I delete this PR then?

File: evennia/contrib/dice.py [hunk 2]
```diff
@@ -169,6 +172,7 @@ def func(self):
         conditional = None
 
         if len_parts < 3 or parts[1] != 'd':
+            branchdata[1].append(0)
             self.caller.msg("You must specify the die roll(s) as <nr>d<sides>."
                             " For example, 2d6 means rolling a 6-sided die 2 times.")
             return
```
Comment (Plongy) [inline evennia/contrib/dice.py]: And here

File: evennia/contrib/dice.py [hunk 3]
```diff
@@ -177,29 +181,41 @@ def func(self):
         ndicelimit = 10000  # Maximum number of dice
         nsidelimit = 10000  # Maximum number of sides
         if int(parts[0]) > ndicelimit or int(parts[2]) > nsidelimit:
+            branchdata[2].append(0)
             self.caller.msg("The maximum roll allowed is %sd%s." % (ndicelimit, nsidelimit))
             return
```
Comment (Plongy) [inline evennia/contrib/dice.py]: And here

**Comment by Plongy** [comment]
Manually merged this

**Pull request closed without merging**